README.rdoc

```diff
@@ -13,11 +13,10 @@ To use, add a flog section to your .codeclimate.yml engines hash:
       flog:
         enabled: true
         config:
-          all: false
-          threshold: 0.6
+          score_threshold: 20.0
 
-You can either toggle :all to true (defaults to false) to report
-everything or tweak the reporting cutoff threshold (defaults to 60%).
+Note: specifying a maximum allowed complexity score is optional.
+The default is 20.0.
```

Comment:
Now that `max_score` was renamed `score_threshold` then maybe this comment needs rewording? "maximum allowed complexity score" seems misleading.